C++/straw.cpp

```diff
@@ -927,12 +927,12 @@ class HiCFile {
     }
 };
 
-long HiCFile::totalFileSize = 0L;
+long long HiCFile::totalFileSize = 0L;
```

Comment:
should this be 0LL (and default in other places as well?)

Reply:
Fixed; thank you.